libopae/plugins/xfpga/plugin.c

```diff
@@ -73,6 +73,9 @@ int __FPGA_API__ opae_plugin_configure(opae_api_adapter_table *adapter,
 {
 	UNUSED_PARAM(jsonConfig);
 
+	if (adapter == NULL || adapter->plugin.dl_handle == NULL)
```

Comment:
These checks aren't needed.  See pluginmgr.c : opae_plugin_mgr_load_dflt_plugins( ).

Reply:
test cases crash/ core dump  if user pass NULL adapter table pointer.

Reply:
Sure, however, I was pointing out that the code which calls the adapter entry point will have already confirmed that adapter is not NULL and that adapter->plugin.dl_handle is not NULL.

Reply:
fixed